dapps/components.vue/src/components/forms/forms.ts

```diff
@@ -57,6 +57,7 @@ export default class Forms extends mixins(EvanComponent) {
   field2 = 'a';
   field3 = 0;
   field4 = 0;
+  field5 = null;
```

Comment:
Can we also add a `v-select` sample to the vue form? Could be something like this:

```
  vSelect: {
    value: '',
    validate: function(vueInstance: Forms, form: SampleFormInterface) {
      return this.value.length !== 0;
    },
    uiSpecs: {
      type: 'v-select',
      attr: {
       options: this.contacts,
       taggable:true
      }
    }
  },
```

Reply:
solved :)